quaint/src/ast/function/row_to_json.rs

```diff
@@ -2,7 +2,6 @@ use super::Function;
 use crate::ast::Table;
 
 #[derive(Debug, Clone, PartialEq)]
-#[cfg_attr(feature = "docs", doc(cfg(feature = "postgresql")))]
 #[cfg(feature = "postgresql")]
```

Comment:
So, this always makes it to `quaint`'s docs, regardless of the feature flag being used or not? Anything I'm missing here?

Reply:
Correct.